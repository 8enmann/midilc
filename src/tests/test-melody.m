/* Test creating a melody
   Tests by creating a simple melody 
   @author Fred
   */

main() {
  Note e;
  Note d;
  Note c;
  Note fc;
  Note fd;
  Sequence b;
  
  e = E4q;
  d = D4q;
  c = C4q;
  fc = C4s;
  fd = D4s;
  
  
  b = new_sequence();
  b = b + e + d + c + e + d + c;
  b = b + fc + fc + fc + fc;
  b = b + fd + fd + fd + fd;
  b = b + e + d + c;
  
  play(b);

}
