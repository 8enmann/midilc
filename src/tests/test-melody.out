0,4,64
4,4,62
8,4,60
12,4,64
16,4,62
20,4,60
24,1,60
25,1,60
26,1,60
27,1,60
28,1,62
29,1,62
30,1,62
31,1,62
32,4,64
36,4,62
40,4,60

